### PR TITLE
MDS - Lesson Three Homework

### PR DESCRIPTION
**Pull request by dudepare**
Done.

### CODE REVIEW

File: timetracker/entries/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+    <head>
+        <title>MelbDjango - Lesson Three</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    </head>
+    <body>
+        <h1>{{ category }}</h1>
```
Comment (funkybob) [inline timetracker/entries/templates/base.html:7]: Is `category` available in every view?

File: timetracker/entries/views.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+def getEntries(client):
+
+    summary = Entry.objects.order_by('project').filter(project__client__name=client)
+    return summary
+
+
+def clientsummary(request):
+
+    clients = Client.objects.all()
+    results = {}
```
Comment (funkybob) [inline timetracker/entries/views.py:29]: Using dicts to pass a set of data for a template is rarely a good idea, as they are unordered.  Instead a list of two-value lists would work just fine.

File: timetracker/entries/templates/base.html (hunk 1 above)
Comment (dudepare) [inline timetracker/entries/templates/base.html:7]: For this homework, yes.

File: timetracker/entries/views.py (hunk 2 above)
Comment (dudepare) [inline timetracker/entries/views.py:29]: Yes I observed that they are not sorted.
so do you mean from:
{ "key1" : [value1, value2, value3], "key2" : [value1, value2] }
 to:
[ [key1, [value1, value2, value3]] , [key2, [value1, value2]] ]

Comment (funkybob) [inline timetracker/entries/views.py:29]: the latter... otherwise you still have a dict.

**Pull request closed without merging**